src/Features/Core/Portable/InlineHints/InlineHint.cs

```diff
@@ -15,19 +15,30 @@ namespace Microsoft.CodeAnalysis.InlineHints
     {
         public readonly TextSpan Span;
         public readonly ImmutableArray<TaggedText> DisplayParts;
+        public readonly TextChange? ReplacementText;
```

Comment:
Replacement or ReplacementChange or ReplacementTextChange or TextChange.  It's no longer just 'Replacement' 'Text'.